tests/monitors/kubernetes_cluster/kubernetes_cluster_test.py

```diff
@@ -275,3 +275,30 @@ def test_deployments(k8s_cluster):
                     },
                 )
             )
+
+
+@pytest.mark.kubernetes
+def test_containers(k8s_cluster):
```

Comment:
Test that containers with limits/requests go through as metrics.  Also test the case where there is a missing limit/request and make sure no metrics are generated for it.

Reply:
Updated the tests to check for this.